Feature/builder

xfn3t opened the pull request.

xfn3t committed:
product with builder

xfn3t committed:
test

xfn3t committed:
entrypoint

xfn3t committed:
git env

xfn3t committed:
git env

xfn3t committed:
change to inner class

gobaz095 reviewed (changes requested):
Те же замечания что и к https://github.com/xfn3t/webbee-homework/pull/8#pullrequestreview-2969460016

xfn3t committed:
refactor!: migrate project to Maven structure

- Added pom.xml with project configuration
- Restructured source code to src/main/java
- Created test directory src/test/java

BREAKING CHANGE: Project now requires Maven for building.
All dependencies must be declared in pom.xml.
Legacy build system is no longer supported.

gobaz095 reviewed (approved):
Ok
